jshw/adventure.js

```diff
@@ -0,0 +1,59 @@
+// Couldn't get all my prompts to work as I wanted but they will get there
+// How to make prompt appear even nothing is entered
+name = prompt('What is your damn name?' + '');
+
+
+//wont run this prompt('Hello ' + name '.' 'Domestic or International?');
+prompt('Domestic or International?')
+var goDomestic = 'domestic';
+var goInternational = 'international';
+// try tripType instead of dom and inter
+
+if (goDomestic === 'domestic') {
+  prompt('What is your budget?')
+} else if (goInternational === 'international') {
+  prompt('What is your budget?????')
+}
+
+
+
+
+
+var budget;
+// wont run if only an if and else section
+if (budget >= 500 && budget <= 999) {
```

Comment:
This will never run since `budget` is undefined from line 22. You need to set it on line 15